calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -4,6 +4,8 @@
 import sys
 from subprocess import check_output, CalledProcessError, check_call
 import requests
+from urllib import quote
+from json import loads as load_json
```

Comment:
I wouldn't mask the name of a module unless you need to - it makes the code harder to follow.  I'd just import json.

json.loads( ) is what people will expect when reading this code.